packages/library/README.md

```diff
@@ -19,9 +19,10 @@ for Web or Node.js targets.
 - Automatically marks dependencies as external
 - Easily extensible to customize your library as needed
 
-**Important! This preset does not include @babel/polyfill for size reasons. If
-you need polyfills in your library code, consider importing @babel/polyfill,
-core-js, or other alternative.**
+\*\*Important! If you need polyfills in your library code, consider including
```

Comment:
I couldn't tell if the `\*\*` here and in the other files changed was a shortform syntax that I wasn't familiar with (that the GitHub diff viewer didn't support, but the docs generator does), or if the intention was to display asterisks without bolding? From looking at the preview it seems that the text isn't bold in the docs:
https://deploy-preview-1484--neutrinojs.netlify.com/packages/library/

I think I slightly prefer the previous styling - though the bold is perhaps slightly overwhelming, so if there was something halfway between that might work? eg Making just the "Important!" bold? Or else using one of the banner styles here:
https://squidfunk.github.io/mkdocs-material/extensions/admonition/#types

Reply:
Yeah not intentional…my editor auto-prettier'd on save and did this. I'll fix.

Reply:
Ah that makes sense! I was thinking it must have been intentional given three files formatted the same hehe